src/certificator/certificator.py

```diff
@@ -30,6 +29,11 @@
 OPA_URL = os.environ.get("F7T_OPA_URL","http://localhost:8181").strip('\'"')
 POLICY_PATH = os.environ.get("F7T_POLICY_PATH","v1/data/f7t/authz").strip('\'"')
 
+### SSL parameters
+USE_SSL = os.environ.get("F7T_USE_SSL", False)
```

Comment:
Should be a cast to boolean instead. Setting `F7T_USE_SSL=False` is not disabling SSL because in the code the flag is being evaluated a string (`if F7T_USE_SSL: ...`)
Either we fix it now or add a task to backlog.

Reply:
We should have a new release after mid-March, so this could be redirected to issues.